src/Ethereum/AddressChecksum.cpp

```diff
@@ -13,12 +13,12 @@
 using namespace TW;
 using namespace TW::Ethereum;
 
-std::string Ethereum::checksumed(const Address& address, enum ChecksumType type) {
+std::string Ethereum::checksumed(const Address& address, [[maybe_unused]] enum ChecksumType type) {
```

Comment:
let's remove `ChecksumType`

Reply:
Agree. Apparently we no longer treat wanchain checksum type separately. The `ChecksumType` enum could be also removed.

Reply:
Handled in #2407